src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticSourceEventSource.cs

```diff
@@ -25,10 +25,22 @@ namespace System.Diagnostics
     ///   
     ///   This EventSource defines a EventSource argument called 'FilterAndPayloadSpecs' that defines
     ///   what DiagnsoticSources to enable and what parts of the payload to serialize into the key-value
-    ///   list that will be forwarded to the EventSource.    If it is empty, all serializable parts of
-    ///   every DiagnosticSource event will be forwarded (this is NOT recommended for monitoring but 
-    ///   can be useful for discovery).
+    ///   list that will be forwarded to the EventSource.    If it is empty, values of propertyies of the 
+    ///   diagnostic source payload are dumped as strings (using ToString() and forwarded to the EventSource.  
+    ///   For what people think of as serialiable object strings, primitives this gives you want you want. 
```

Comment:
Nit: I'm not understanding the grammar of this sentence... is it meant to be "For what people think of as serializable objects, strings, and primitives, this gives you what you want."